rules/internal/utils/common_lints_test.go

```diff
@@ -227,3 +228,36 @@ func TestLintMethodHasMatchingResponseName(t *testing.T) {
 		})
 	}
 }
+
+func TestLintFieldProperties(t *testing.T) {
```

Comment:
See other comment:

1. Just test `LintOneofField`
2. Test newly exposed method, `LintSingularField`

Reply:
Done. I went with `LintNotOneof` because of the "assertive" style of the other functions (e.g. `LintSingularField` "asserts" that the field is not repeated).